src/hayhooks/cli/deploy_files/__init__.py

```diff
@@ -2,23 +2,23 @@
 import requests
 from pathlib import Path
 from urllib.parse import urljoin
-from hayhooks.server.utils.deploy_utils import read_pipeline_files_from_folder
+from hayhooks.server.utils.deploy_utils import read_pipeline_files_from_dir
 
 
 @click.command()
 @click.pass_obj
 @click.option('-n', '--name', required=True, help="Name of the pipeline to deploy")
-@click.argument('folder', type=click.Path(exists=True, file_okay=False, dir_okay=True, path_type=Path))
-def deploy_files(server_conf, name, folder):
-    """Deploy all pipeline files from a folder to the Hayhooks server."""
+@click.argument('pipeline_dir', type=click.Path(exists=True, file_okay=False, dir_okay=True, path_type=Path))
+def deploy_files(server_conf, name, pipeline_dir):
```

Comment:
Shall we add types here?

Reply:
Yes we can, but to tell you the truth I was planning to add `fastapi[standard]` as a dep and then rewrite the CLI commands using [Typer](https://typer.tiangolo.com/), which is cleaner and has a stronger types support. I've already written it actually, but wanted to tackle those other refactoring first 😉